frameworks/Java/vertx/src/main/java/vertx/WebServer.java

```diff
@@ -75,14 +82,13 @@ public void stop(){
 
 	private void handlePlainText(HttpServerRequest request) {
 		request.response()
-		.putHeader(HEADER_CONTENT, RESPONSE_TYPE_PLAIN).putHeader(HEADER_SERVER,  SERVER)
-		.putHeader(HEADER_DATE, dateString).end(HELLO_WORLD_BUFFER);
+		.putHeader(CONTENT_TYPE , RESPONSE_TYPE_PLAIN).putHeader(SERVER, VERTX)
+		.putHeader(DATE, dateString).putHeader(CONTENT_LENGTH, HELLO_WORLD_CONTENT_LENGTH).end(HELLO_WORLD_BUFFER);
 	}
 
 	private void handleJson(HttpServerRequest request) {
-		Buffer buff = Buffer.buffer(Json.encode(Collections.singletonMap(TEXT_MESSAGE, HELLO_WORLD)));
-		request.response().putHeader(HEADER_CONTENT, RESPONSE_TYPE_JSON).putHeader(HEADER_SERVER,  SERVER)
-		.putHeader(HEADER_DATE, dateString).end(buff);
+		request.response().putHeader(CONTENT_TYPE, RESPONSE_TYPE_JSON).putHeader(SERVER,  VERTX)
+		.putHeader(DATE, dateString).putHeader(CONTENT_LENGTH, JSON_HELLO_WORLD_CONTENT_LENGTH).end(JSON_HELLO_WORLD_BUFFER);
```

Comment:
@whitingjr 
> Otherwise the benchmark results published by TechEmpower are less useful than they could be.
The reason why we write the rules is to do our best to create a level playing field for every project. Obviously, we can miss things. This is an open source project so we rely on the community to help vet the frameworks, exactly like has happened here. If you feel like any framework is in violation of any of the rules we have written, please feel free to open an issue, or a pull request to address it.

We don't have a favorite here. We want a level playing field. However, we also rely on community support. We're not going to make everyone happy. We understand that this project is wide-ranging in scope. We don't believe that there is a `winner` of the benchmarks. We want you to take from it what you can; hopefully a way to help you improve your framework and help others improve theirs.